README.md

```diff
@@ -1,70 +1,298 @@
+   </td>
+  </tr>
+  <tr>
+   <td>Title
+   </td>
+   <td>The title for the payment method in the checkout page
+   </td>
+  </tr>
+  <tr>
+   <td>Custom Expiry
+   </td>
+   <td>This field will allow you to set custom duration on how long the transaction available to be paid.
+   </td>
+  </tr>
+  <tr>
+   <td>Sort Order
+   </td>
+   <td>The function do to sort your payment method in the checkout page
+   </td>
+  </tr>
+  <tr>
+   <td>Allowed Payment Method \
+   </td>
+   <td>Customize allowed payment method, separate payment method code with a comma. e.g: bank_transfer,credit_card. Leave it default if you are not sure.
+   </td>
+  </tr>
+  <tr>
+   <td>Acquiring Bank
+   </td>
+   <td>You can specify which Acquiring Bank they prefer to use for a specific transaction. The transaction fund will be routed to that specific acquiring bank. Leave it blank if you are not sure!
+   </td>
+  </tr>
+  <tr>
+   <td>BIN Number
+   </td>
+   <td>It is a feature that allows the merchant to accept only Credit Cards within a specific set of BIN numbers. Separate BIN number with comma Example: 4,5,4811,bni,Mandiri. Leave it blank if you are not sure!
+   </td>
+  </tr>
+  <tr>
+   <td>Installment Terms<br>
+<p><small>
+Only available in Offline payment specific</small>
+   </td>
+   <td>an arrangement for payment by installments.
+   </td>
+  </tr>
+  <tr>
+   <td>3D Secure
+   </td>
+   <td>You must enable 3D Secure for secure card transactions. Please contact us if you wish to disable this feature in the Production environment.
+   </td>
+  </tr>
+  <tr>
+   <td>Save Card
+   </td>
+   <td>This will allow your customer to save their card on the payment popup, for faster payment flow on the following purchase.
+   </td>
+  </tr>
+</table>
+
+
+
+## Magento Midtrans Notifications Handling
+
+Login to your [Midtrans Account](https://dashboard.midtrans.com/settings/vtweb_configuration), select your environment (sandbox/production), go to menu **settings** > **configuration** and feel out the following URL fields:
+
+
+
+*   Payment Notification URL:
+
+    `https://[your-site-url]/snap/payment/notification`
+
+*   Finish Redirect URL:
+
+    `https://[your-site-url]/snap/index/finish`
+
+*   Unfinish Redirect URL:
+
+    `https://[your-site-url]/snap/index/finish`
+
+*   Error Redirect URL:
+
+    `https://[your-site-url]/snap/index/finish`
+
+
+## How to online refund transaction
+
+You can request refunds either in the [Midtrans Dashboard](https://dashboard.midtrans.com/transactions) or in the Magento admin. After a refund is issued, it cannot be cancelled or undone. Before you trigger this request, make sure that the refund amount and any other details are correct. The online refund function is available only for payment method gopay and credit card.
+
+If you make a refund in the Midtrans Dashboard, a Refund notification is sent to Magento, set transaction state to CLOSE and for now is not created the credit memo.
```

Comment:
> refund

not
> refund

> from the

Reply:
unclear:
> set transaction state to CLOSE and for now is not created the credit memo.

please clarify. "it will change transaction state to **Close**?

Reply:
close or closed?

Reply:
@Xaxxis reminder